extensions/kube-object-event-status/package-lock.json

```diff
@@ -6,7 +6,691 @@
   "dependencies": {
     "@k8slens/extensions": {
       "version": "file:../../src/extensions/npm/extensions",
-      "dev": true
+      "dev": true,
```

Comment:
Why so big diff? 🤔  Different npm version used?

Reply:
Oh bother these keep slipping in.

Reply:
Might be that but also I am changing our package contents.